PowerEditor/src/WinControls/TabBar/TabBar.cpp

```diff
@@ -643,16 +655,9 @@ LRESULT TabBarPlus::runProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lPara
 				}
 				else if (++_dragCount > 2)
 				{
-					int tabFocused = static_cast<int32_t>(::SendMessage(_hSelf, TCM_GETCURFOCUS, 0, 0));
 					int tabSelected = static_cast<int32_t>(::SendMessage(_hSelf, TCM_GETCURSEL, 0, 0));

```

Comment:
Just in case, put everything below it into a conditional block:

`if (tabSelected != -1)`

Reply:
Sure.